packages/hoppscotch-common/src/components/app/Header.vue

```diff
@@ -379,6 +393,24 @@ const handleTeamEdit = () => {
   }
 }
 
+const deleteTeam = () => {
+  if (!teamID.value) return
+  pipe(
+    backendDeleteTeam(teamID.value),
+    TE.match(
+      (err) => {
+        // TODO: Better errors ? We know the possible errors now
+        toast.error(`${t("error.something_went_wrong")}`)
+        console.error(err)
```

Comment:
Does the GQL mutation return a user-friendly error message that can help with more context to the user?

Also, is `console.error` intentional?